packages/adena-extension/src/pages/popup/wallet/approve-transaction-main/index.tsx

```diff
@@ -96,6 +96,10 @@ const ApproveTransactionContainer: React.FC = () => {
   }, [document]);
 
   const isErrorNetworkFee = useMemo(() => {
+    //skip for case sign msg for sponsor service
+    if (document?.msgs[0].type === '/vm.m_noop') {
+      return false;
+    }
```

Comment:
You need to check for multi-messages.
Additionally, it would be nice to have a separate type-checking function for this.

Reply:
we separate type-checking function and add condition to check for multi-messages.

Reply:
the changes address at https://github.com/onbloc/adena-wallet/pull/540/commits/764ad3e5bfc0458907955be914bc01b091e22859